layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "shaderUniformBufferArrayNonUniformIndexingNative": 0,
+        "shaderSampledImageArrayNonUniformIndexingNative": 0,
+        "shaderStorageBufferArrayNonUniformIndexingNative": 0,
+        "shaderStorageImageArrayNonUniformIndexingNative": 0,
+        "shaderInputAttachmentArrayNonUniformIndexingNative": 0,
+        "robustBufferAccessUpdateAfterBind": 0,
+        "quadDivergentImplicitLod": 0,
+        "maxPerStageDescriptorUpdateAfterBindSamplers": 500000,
+        "maxPerStageDescriptorUpdateAfterBindUniformBuffers": 12,
+        "maxPerStageDescriptorUpdateAfterBindStorageBuffers": 500000,
+        "maxPerStageDescriptorUpdateAfterBindSampledImages": 500000,
+        "maxPerStageDescriptorUpdateAfterBindStorageImages": 500000,
+        "maxPerStageDescriptorUpdateAfterBindInputAttachments": 4,
+        "maxPerStageUpdateAfterBindResources": 500000,
+        "maxDescriptorSetUpdateAfterBindSamplers": 500000,
+        "maxDescriptorSetUpdateAfterBindUniformBuffers": 72,
+        "maxDescriptorSetUpdateAfterBindUniformBuffersDynamic": 8,
+        "maxDescriptorSetUpdateAfterBindStorageBuffers": 500000,
+        "maxDescriptorSetUpdateAfterBindStorageBuffersDynamic": 4,
+        "maxDescriptorSetUpdateAfterBindSampledImages": 500000,
+        "maxDescriptorSetUpdateAfterBindStorageImages": 500000,
+        "maxDescriptorSetUpdateAfterBindInputAttachments": 4
+    },
+    "VkPhysicalDeviceFloatControlsProperties": {
+        "denormBehaviorIndependence": 2,
+        "roundingModeIndependence": 2,
+        "shaderSignedZeroInfNanPreserveFloat16": 0,
+        "shaderSignedZeroInfNanPreserveFloat32": 0,
+        "shaderSignedZeroInfNanPreserveFloat64": 0,
+        "shaderDenormPreserveFloat16": 0,
+        "shaderDenormPreserveFloat32": 0,
+        "shaderDenormPreserveFloat64": 0,
+        "shaderDenormFlushToZeroFloat16": 0,
+        "shaderDenormFlushToZeroFloat32": 0,
+        "shaderDenormFlushToZeroFloat64": 0,
+        "shaderRoundingModeRTEFloat16": 0,
+        "shaderRoundingModeRTEFloat32": 0,
+        "shaderRoundingModeRTEFloat64": 0,
+        "shaderRoundingModeRTZFloat16": 0,
+        "shaderRoundingModeRTZFloat32": 0,
+        "shaderRoundingModeRTZFloat64": 0
+    },
+    "VkPhysicalDeviceMaintenance3Properties": {
+        "maxPerSetDescriptors": 1024,
+        "maxMemoryAllocationSize" : 536870912
+    },
+    "VkPhysicalDeviceMultiviewProperties": {
+        "maxMultiviewViewCount": 6,
+        "maxMultiviewInstanceIndex" : 67108863
+    },
+    "VkPhysicalDeviceProtectedMemoryProperties": {
+        "protectedNoFault": 0
+    },
+    "VkPhysicalDeviceSamplerFilterMinmaxProperties": {
+        "filterMinmaxSingleComponentFormats": 0,
+        "filterMinmaxImageComponentMapping": 0
+    },
+    "VkPhysicalDeviceTimelineSemaphoreProperties": {
+        "maxTimelineSemaphoreValueDifference": 1073741823
+    },
+    "VkPhysicalDeviceFeatures": {
+        "robustBufferAccess": 1,
```

Comment:
Enabled by default? 
Also, robustness is one of the rare feature that as a high cost to be enabled.

This said, I can see how this could be useful during development to ensure the application is correct. Is this why it's enabled?

Reply:
This is not enabled, this is saying that the feature is available when you call `vkGetPhysicalDeviceFeatures*`.
The user can then try to enable it.
Since this is simulation and not emulation it will only work if the underlying device has this feature, which it should since it is a required feature for Vulkan 1.2.